map/map_loader/README.md

```diff
@@ -174,10 +180,7 @@ lanelet2_map_visualization visualizes autoware_auto_mapping_msgs/HADMapBin messa
 
 ### Parameters
 
-| Name                        | Type        | Description                                                  | Default value |
-| :-------------------------- | :---------- | :----------------------------------------------------------- | :------------ |
-| lanelet2_map_projector_type | std::string | The type of the map projector using, can be MGRS, UTM, local | MGRS          |
-| latitude                    | double      | Latitude of map_origin, only using in UTM map projector      | 0.0           |
-| longitude                   | double      | Longitude of map_origin, only using in UTM map projector     | 0.0           |
-| center_line_resolution      | double      | Define the resolution of the lanelet center line             | 5.0           |
-| lanelet2_map_path           | std::string | The lanelet2 map path                                        | None          |
+| Name                   | Type        | Description                                      | Default value |
+| :--------------------- | :---------- | :----------------------------------------------- | :------------ |
+| center_line_resolution | double      | Define the resolution of the lanelet center line | 5.0           |
+| lanelet2_map_path      | std::string | The lanelet2 map path                            | None          |
```

Comment:
These should be moved under `lanelet2_map_loader`

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3986/commits/4b8ddbbb0bd9f0870cd822993b0d3fc44c866fbe